NetKAN/Workshop.netkan

```diff
@@ -34,6 +34,17 @@
                     { "name": "KIS" }
                 ]
             }
+        },
+	{	
+            "version": ">=0.11.0",
+            "override": {
+                "depends": [
+                    { "name": "ModuleManager" },
+                    { "name": "CommunityResourcePack", "minVersion": "0.4.7" },
+		    { "name": "Firespitter Core" },
```

Comment:
ok i will fix that, too
Am 17.11.2015 16:34 schrieb "Arne" <notifications@github.com>:

> In NetKAN/Workshop.netkan
> <https://github.com/KSP-CKAN/NetKAN/pull/2615#discussion_r45073880>:
>
> > @@ -34,6 +34,17 @@
> >                      { "name": "KIS" }
> >                  ]
> >              }
> > +        },
> > +	{	
> > +            "version": ">=0.11.0",
> > +            "override": {
> > +                "depends": [
> > +                    { "name": "ModuleManager" },
> > +                    { "name": "CommunityResourcePack", "minVersion": "0.4.7" },
> > +		    { "name": "Firespitter Core" },
>
> No, it needs to be "min_version": "0.4.7.0" for CRP
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KSP-CKAN/NetKAN/pull/2615/files#r45073880>.
>